packages/react/src/components/TableOfContents/__stories__/TableOfContents.stories.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TABLE_OF_CONTENTS } from '../../../internal/FeatureFlags';
```

Comment:
@sannminwin need to update this one: `DDS_TOC`

Reply:
hey Jeff,
correct me if I am wrong. 
We are export as TABLE_OF_CONTENTS from FeatureFlads.js right ?

`export const TABLE_OF_CONTENTS =
  process.env.DDS_TOC === 'true' || DDS_FLAGS_ALL || false;
`

Reply:
updated.